tests/python_tests/test_rag.py

```diff
@@ -332,6 +333,53 @@ def test_qwen3_embedding(download_and_convert_embeddings_models, dataset_documen
     validate_embedding_results(embeddings_genai, embeddings_opt.tolist())
 
 
+@pytest.mark.parametrize("download_and_convert_embeddings_models", ["Qwen/Qwen3-Embedding-0.6B"], indirect=True)
+@pytest.mark.parametrize(
+    "config",
+    [
+        TextEmbeddingPipeline.Config(normalize=False, pooling_type=TextEmbeddingPipeline.PoolingType.LAST_TOKEN, padding_side="left", batch_size=1),
+        TextEmbeddingPipeline.Config(normalize=False, pooling_type=TextEmbeddingPipeline.PoolingType.LAST_TOKEN, batch_size=1),
+    ],
+)
+@pytest.mark.precommit
+@pytest.mark.xfail(condition=(sys.platform == "darwin"), reason="Ticket - 174635")
+def test_qwen3_embedding_set_batch(download_and_convert_embeddings_models, dataset_documents, config):
```

Comment:
Let's extend already implemented `test_qwen3_embedding` instead of introducing new test